README.md

```diff
@@ -5,11 +5,38 @@ Create SaaS Boilerplate
 [![Downloads/week](https://img.shields.io/npm/dw/create-saas-boilerplate.svg?style=for-the-badge)](https://npmjs.org/package/create-saas-boilerplate)
 [![License](https://img.shields.io/npm/l/create-saas-boilerplate.svg?style=for-the-badge)](https://github.com/apptension/create-saas-boilerplate/blob/master/package.json)
 
-<!-- toc -->
-* [Usage](#usage)
-* [Commands](#commands)
-<!-- tocstop -->
-# Usage
+Create SaaS Boilerplate is a CLI tool designed to quickly set up a new local instance of the SaaS Boilerplate. It is a specialized tool that allows users to get started with the SaaS Boilerplate as quickly as possible. 
+
+---
+
+📖 [**SaaS Boilerplate Documentation**](https://docs.demo.saas.apptoku.com/)
+
+🌟 [**SaaS Boilerplate Page**](https://www.apptension.com/saas-boilerplate)
+
+---
+
+## Initialize new SaaS Boilerplate project
+
+```
+USAGE
+  $ yarn create saas-boilerplate PATH [--skipSystemCheck]
+  $ pnpm create saas-boilerplate PATH [--skipSystemCheck]
+  $ npm init saas-boilerplate PATH [--skipSystemCheck]
+ARGUMENTS
+  PATH  [default: .] Directory name where to initialize project
+FLAGS
+  --skipSystemCheck  Skip system check
+DESCRIPTION
+  Initialize new SaaS Boilerplate project
+```
+_See code: [dist/commands/init/index.ts](https://github.com/apptension/create-saas-boilerplate/blob/v0.0.1/dist/commands/init/index.ts)_
+
+## Usage
```

Comment:
@mkleszcz I've added the usage description above the `<!-- usage -->` tag so that it won't be overridden.

Reply:
Thanks, but I don't think anyone will install this tool globally tbh ;) You can remove the description below.
Basically the 99% of the usage will be calling `yarn create saas-boilerplate` and if someone will install this globally will know how to use it;)

Reply:
Should I remove this whole Usage block? I believe that `Initialize new SaaS Boilerplate project` section should be enough

Reply:
ok... you can remove the entire `Usage` section. We'll keep it simple:)